src/node.cc

```diff
@@ -3478,16 +3478,37 @@ Local<Context> NewContext(Isolate* isolate,
   auto context = Context::New(isolate, nullptr, object_template);
   if (context.IsEmpty()) return context;
   HandleScope handle_scope(isolate);
-  auto intl_key = FIXED_ONE_BYTE_STRING(isolate, "Intl");
-  auto break_iter_key = FIXED_ONE_BYTE_STRING(isolate, "v8BreakIterator");
   context->SetEmbedderData(
       ContextEmbedderIndex::kAllowWasmCodeGeneration, True(isolate));
+
+  auto intl_key = FIXED_ONE_BYTE_STRING(isolate, "Intl");
```

Comment:
Is there a reason these lines got moved?

Reply:
just a side-effect of the changes i was making before i finished up. should i change it back? they kinda make the order better anyway.